README.md

```diff
@@ -1,3 +1,43 @@
+\frontmatter
+
 # Unfug für Anfänger
 
-_to be written_
+*Unfug für Anfänger* ist ein Buch für alle die an Informatik interessiert sind.
+Es hat mehrere Aufgaben zu erfüllen:
+
+- Referenz für Neueinsteiger in die Welt des modernen Programmierens
+- Neue Ideen für diverse Themen die Praktisch oder einfach nur cool sind
+- Eine FAQ sein
+
+Ob es diese auch schafft bleibt den Lesern überlassen. Humor ist erlaubt und
+auch erwünscht, am Ende geht es darum Spass zu haben; am Programmieren, an der
+Technik, an der Zukunft.
+
+Das Buch ist grob in Teile aufgebaut um den anschein einer Ordnung zu bewahren.
+Das Inhaltsverzeichniss wird bestimmt helfen das zu finden wonach man sucht.
+
+Mit dem sagen wir: Viel Spass beim lesen und treibt ja _UnFUG_.
```

Comment:
`Mit dem sagen wir`?

Ausserdem: `treibt ja _UnFUG_` würde ich in `treibt ja schön viel _UnFUG_` ändern,